packages/neuron-wallet/package.json

```
@@ -42,13 +42,13 @@
     "@iarna/toml": "2.2.5",
     "@ledgerhq/hw-transport-node-hid": "6.27.13",
     "@nervina-labs/ckb-indexer": "0.1.1",
+    "@nervosnetwork/neuron-shared": "0.110.1",
     "@nervosnetwork/ckb-sdk-core": "0.107.0",
     "@nervosnetwork/ckb-sdk-utils": "0.107.0",
```

```suggestion
    "@nervosnetwork/ckb-sdk-core": "0.107.0",
    "@nervosnetwork/ckb-sdk-utils": "0.107.0",
    "@nervosnetwork/neuron-shared": "0.110.1",
```

Dependencies managed by `lerna` are ordered alphanumerically.